MergePythonSDK/hris/model/employee.py

```diff
@@ -428,40 +448,40 @@ def __init__(self, *args, **kwargs):  # noqa: E501
         self._configuration = _configuration
         self._visited_composed_classes = _visited_composed_classes + (self.__class__,)
 
-        self.remote_id = kwargs.get("remote_id", None)
-        self.employee_number = kwargs.get("employee_number", None)
-        self.company = kwargs.get("company", None)
-        self.first_name = kwargs.get("first_name", None)
-        self.last_name = kwargs.get("last_name", None)
-        self.display_full_name = kwargs.get("display_full_name", None)
-        self.username = kwargs.get("username", None)
-        self.groups = kwargs.get("groups", list())
-        self.work_email = kwargs.get("work_email", None)
-        self.personal_email = kwargs.get("personal_email", None)
-        self.mobile_phone_number = kwargs.get("mobile_phone_number", None)
-        self.employments = kwargs.get("employments", list())
-        self.home_location = kwargs.get("home_location", None)
-        self.work_location = kwargs.get("work_location", None)
-        self.manager = kwargs.get("manager", None)
-        self.team = kwargs.get("team", None)
-        self.pay_group = kwargs.get("pay_group", None)
-        self.ssn = kwargs.get("ssn", None)
-        self.gender = kwargs.get("gender", None)
-        self.ethnicity = kwargs.get("ethnicity", None)
-        self.marital_status = kwargs.get("marital_status", None)
-        self.date_of_birth = kwargs.get("date_of_birth", None)
-        self.hire_date = kwargs.get("hire_date", None)
-        self.start_date = kwargs.get("start_date", None)
-        self.remote_created_at = kwargs.get("remote_created_at", None)
-        self.employment_status = kwargs.get("employment_status", None)
-        self.termination_date = kwargs.get("termination_date", None)
-        self.avatar = kwargs.get("avatar", None)
-        self.custom_fields = kwargs.get("custom_fields", None)
+        self.remote_id: Optional[str, none_type] = kwargs.get("remote_id", None)
```

Comment:
`Optional[str, none_type]` is wrong since `Optional[x]` automatically means x or none. If it's easier to keep `none_type` as an explicit option then use `Union` but i'd prefer just `Optional[str]` here for cleanliness.